pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_delete-modules.adoc

```diff
@@ -0,0 +1,18 @@
+[id="delete-modules_{context}"]
+
+= Deleteing a module in {ProductShortName}
+
+Deleting a module in {ProductShortName}.
+
+.Prerequisites
+
+//* You are assigned a publisher or admin role in {ProductShortName}.
+* You are logged in.
+
+.Procedure
+
+. Navigate to the link:{LinkToSearchPage}[Search page].
+. Select the check box of the module you want to delete.
```

Comment:
would it be better to just say "Select the module you want to delete"?

Reply:
I don't think it is clear to the user that they need to actually check the box.